ui/app/components/tools/wrap.js

```diff
@@ -24,15 +25,31 @@ export default class ToolsWrap extends Component {
   @tracked buttonDisabled = false;
   @tracked token = '';
   @tracked wrapTTL = null;
-  @tracked wrapData = '{\n}';
+  @tracked wrapData;
   @tracked errorMessage = '';
+  @tracked showJson = true;
```

Comment:
nit: can you add an extra line here to create some visual space between the `@tracked` variables and getters below?